admin/student_profile.php

```diff
@@ -0,0 +1,56 @@
+<?php
+include('../include/initialize.php');
+
+$tittle = "Student Profile";
+echoHeader($tittle); 
+echoVerticalBar();
+echoHorizontalBar($tittle, "");
+
+if(isset($_REQUEST['id'])){
+
+    $studentId = $_REQUEST['id'];                               
+    $student = getStudent($studentId);
+    
+    if(!empty($student)){     
+        echoProfileInfo($student, 'bishop');                                             //will handle profiles pictures later
+    }                                                        
+                   
+    else{
+        echo"No student!";                                                               //This will be used properly later on(e.g. showing the correct message etc)
+    }         
+    
+    echoFormButtons(calcStudentRemainingCredits($studentId));
+    
+    echo"<div class='flex-container-classesInfo'>";
+            echoClassesInfo(getFutureClassesbyStudent($studentId), 'Classes Booked');
+            echoClassesInfo(getStudentPastClasses($studentId), 'Recent Classes');
```

Comment:
This is a minor function design thing, but I'm curious why you chose to pass `studentId` for these two functions instead of a full `student` object. There isn't a right or wrong answer here, and it's something that we generally mix and match in our own code base. There are pros and cons to each (passing the id makes the function easier to work with if you don't already have the full object; passing the object can save you on 1 or more DB queries; etc), so I'm just interested in hearing what your thought process was here.

Reply:
If I'm misunderstanding the question let me know. So I think in this case the student object does not have the class information I need since I'm looking for class information that it is not in the student table. So, getFutureClassesbyStudent($studentId) and getStudentPastClasses($studentId) return a class object that it is what I'm passing to echoClassesInfo to echo the class information (I only echo the date tho). 
**Here is an instance of what student has:**
array(10) {
  ["StudentId"]=>
  int(1)
  ["FirstName"]=>
  string(5) "Diany"
  ["LastName"]=>
  string(5) "Leyva"
  ["Email"]=>
  string(18) "diany9012@yahoo.es"
  ["Phone"]=>
  string(12) "305-569-3858"
  ["ELO"]=>
  int(0)
  ["LichessLink"]=>
  string(11) "lichess.com"
  ["DateCreated"]=>
  string(19) "2022-06-08 09:04:37"
  ["PrivateNotes"]=>
  string(30) "No previous chess experience  "
  ["PublicNotes"]=>
  string(0) ""
}

**And here is what getFutureClassesbyStudent($studentId) returns:**
array(1) {
  [0]=>
  array(6) {
    ["ClassId"]=>
    int(2)
    ["Type"]=>
    string(6) "Online"
    ["ZoomLink"]=>
    string(22) "https://zoom.us/signin"
    ["StartDate"]=>
    string(19) "2022-06-30 12:00:00"
    ["Duration"]=>
    int(60)
    ["StudentId"]=>
    int(1)
  }
}